package.json

```diff
@@ -14,7 +14,8 @@
     "lib"
   ],
   "scripts": {
-    "build": "vite build"
+    "build": "vite build",
+    "postinstall": "husky install"
```

Comment:
Husky moet op de `prepare` hook, niet `postinstall`, en je kunt het beste het pakketje `is-ci` erbij gebruiken, anders krijg je nog steeds hetzelfde probleem.

```shell
yarn add -D is-ci
```

```json
"prepare": "is-ci || husky install"
```

https://typicode.github.io/husky/#/?id=with-is-ci